### ISSUE TITLE
Killing mobs with your pet, without you dealing any damage should not give xp or ability to loot mob

### ISSUE THREAD

**Issue by karabitaz** [Atlantiss/NetherwingBugtracker#3190]
**Description**: when you send your pet to kill a mob, and don't do any damage yourself, mob still gives you xp and ability to loot it.

**Current behaviour**: mob gives you xp

**Expected behaviour**: mob shouldnt give you xp, https://www.youtube.com/watch?v=pJcXFjwj4QQ in this video you can see lock killing mobs with his felguard , and only hitting once with wand so that he would get xp and loot.

**Server Revision**: 2382

**Comment by karabitaz** [comment]
Yes, this is tbc beta, but it was not changed after beta. Mobs never gave xp or loot in retail if it was killed solo by pet.

**Comment by Godmatik** [comment]
Rev 4017 - still an issue
